flow/directives.py

```diff
@@ -99,8 +99,7 @@ def validate_callable(*jobs):
                 return self._validator(value(*jobs))
 
             return validate_callable
-        else:
```

Comment:
I don't have a problem with this, but having talked to other developers. Whether including the `else` block here is a code-smell or whether not including it is not well agreed upon. I tend to think the else block helps, but I don't feel strongly either way.

Reply:
@b-butler, pylint suggested all of these changes that remove dangling else clauses and the like. I thought about it for a while and concluded that it was a justifiable (though minor) change.

Reply:
I’m not strongly opinionated on it, but I do like that it expresses the same logic at a lower indentation level, in fewer lines, and with less possibility of future code changes introducing a bug (e.g. leaving an “elif” at the end instead of an else) and returning None.

Reply:
If a tool like pylint suggested it, then I am good with the changes.